section2/reflection.md

```diff
@@ -1,15 +1,27 @@
 ## Section 2 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+* From the article, I like the **SMART** (Setting Specific, Measurable, Achievable, Relevant, and Time-Bound) acronym in setting up achievable goals in your work. Often I'll just grind with no (real) goal than to just get the work done, but setting up smaller and more specific goals the SMART way, seems like it would garner more success.
+* From the video, my biggest takeaway is not comparing yourself to anyone but ***yourself***. Other people are on different paths and learn in different ways; sometimes quicker than you can. As long as you're making progress, you're growing, and you'll get to where you want to be.
```

Comment:
LOVE that you pulled this out. This can be hard since so many of our educational experiences have conditioned us to do that. Keep this in mind during your time at Turing (and beyond).